data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -68,7 +68,7 @@ val u : int list = [1; 2; 3; 4]
 - : string list = ["this"; "is"; "mambo"]
 ```
 
-The lists' types, `int list` and `string list`, have been inferred from the type of their elements. Lists can be empty `[]` (pronounced “nil”). Note that the first list has been given a name using the `let … = …` construction, which is detailed below. The most primitive operation on lists is appending a new element at the front of an existing list. This is done using the “cons” operator, written with the double colon operator `::`.
+The lists' types, `int list` and `string list`, have been inferred from the type of their elements. Lists can be empty `[]` (pronounced “nil”). Note that the first list has been given a name using the `let … = …` construction, which is detailed below. The most primitive operation on lists is prepending a new element at the front of an existing list. This is done using the “cons” operator, written with the double colon operator `::`.
```

Comment:
> The most primitive operation on lists is prepending a new element at the front of an existing list.

Replacing *appending* with *prepending* sounds good. However, isn't the second part of the sentence redundant if we do that? Also, isn't _preprend_ a more advanced term? The text needs to be exact and easy to understand.

Reply:
Agreed about it being a more advanced term. 
How about: 

The most primitive operation on lists is to add a new element at the front of an existing list.